cmd/codegen/main.go

```diff
@@ -47,7 +47,7 @@ func ClientGen(cmd *cobra.Command, args []string) error {
 	var dialErr error
 	if propagateLogs {
 		progW, dialErr = progrock.DialRPC(ctx, "unix://"+nestedSock)
-		if err != nil {
+		if dialErr != nil {
 			return fmt.Errorf("error connecting to progrock: %w", err)
```

Comment:
```suggestion
			return fmt.Errorf("error connecting to progrock: %w", dialErr)
```
This one too, right?

Reply:
Good catch! There was another one gotcha down that function. LGTM now.

Reply:
Thank you!